compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/CodeGenerator.java

```diff
@@ -95,16 +91,47 @@ private CompiledJarFile generate(BPackageSymbol packageSymbol) {
         populateExternalMap(jvmPackageGen);
 
         //Rewrite identifier names with encoding special characters
-        HashMap<String, String> originalIdentifierMap = JvmDesugarPhase.encodeModuleIdentifiers(packageSymbol.bir);
+        JvmDesugarPhase.encodeModuleIdentifiers(packageSymbol.bir);
 
         // TODO Get-rid of the following assignment
         packageSymbol.compiledJarFile = jvmPackageGen.generate(packageSymbol.bir, true);
-
-        //Revert encoding identifier names
-        JvmDesugarPhase.replaceEncodedModuleIdentifiers(packageSymbol.bir, originalIdentifierMap);
```

Comment:
We can't remove this line 104. I will send a commit to revert it. Though all the tests when I compile some packages for the second time, an error is thrown at BIRSymbolEnter. We need to either change BIRSymbolEnter + BIRBinaryWriter or replace the encoded identifiers.